packages/calling-component-bindings/review/calling-component-bindings.api.md

```diff
@@ -209,12 +209,18 @@ export const optionsButtonSelector: reselect.OutputParametricSelector<CallClient
 }>;
 
 // @public (undocumented)
-export const participantListSelector: reselect.OutputParametricSelector<CallClientState, CallingBaseSelectorProps, {
```

Comment:
We still expose a `participantList` components, is it worth keeping this export for that component?